examples/pets/tests/createAndShare.spec.ts

```diff
@@ -27,6 +29,8 @@ test("create a new post and share", async ({ page }) => {
 
     const invitation = await postPage.getShareLink();
 
+    await sleep(1000);
```

Comment:
I've reduced the image size and added a sleep to see if the problem was that the "creator" peer didn't have the time to upload the data.